minio/minio
# skip healing properly in the scanner when a drive is hotplugged

## Community Contribution License
All community contributions in this pull request are licensed to the project maintainers
under the terms of the [Apache 2 license](https://www.apache.org/licenses/LICENSE-2.0). 
By creating this pull request I represent that I have the right to license the 
contributions to the project maintainers under the Apache 2 license.

## Description
skip healing properly in the scanner when a drive is hotplugged

## Motivation and Context
due to how the state is passed around the SkipHealing
might not be the true state() of the system always, causing
a situation where we might heal from the scanner on the
a same drive which is being. Due to this, competing heals get
triggered that slow each other down.

## How to test this PR?
You need a setup with four drives minimum and hotplug
replace the drive at random; you can see competing 
heals invoked.

## Types of changes

## Checklist:
